tm2py/emme/matrix.py

```diff
@@ -262,3 +262,91 @@ def read_hdf5(self, path: str) -> NumpyArray:
             Numpy array from OMX file
         """
         return self._omx_file.get_node(path).read()
+
+
+class TollChoiceCalculator:
```

Comment:
why is this in emme/matrix? Does it use Emme?

Reply:
i.e. why isn't this a model component which could then reference some general matrix/choice model language/tools